python/ray/data/dataset.py

```diff
@@ -825,13 +825,13 @@ def aggregate(self, *aggs: Tuple[AggregateFn]) -> U:
 
         Examples:
             >>> ray.data.range(100).aggregate(Max())
-            >>> ray.data.range_arrow(100).aggregate(Max("value"))
+            >>> ray.data.range_arrow(100).aggregate(
+                Max("value"), Mean("value"))
```

Comment:
Should we mark our built-in agg as part of the PublicAPI as well? (add to docs, etc.?)

Reply:
Definitely! I'm planning on adding that in the third stacked PR, since we'll have the full API at that point (nicest UX).